tests/OcrTestCase.php

```diff
@@ -13,10 +13,14 @@ class OcrTestCase extends KernelTestCase
     /** @var string */
     protected $accessToken;
 
+    /** @var string */
+    protected $refreshToken;
+
     protected function setUp(): void
     {
         self::bootKernel();
         $this->projectDir = self::$kernel->getProjectDir();
         $this->accessToken = "eyJhbGciOiJSUzI1NiIsInR5cC";
+        $this->refreshToken = "eyJhbGciOiJSUzI1NiIsInR5cC";
```

Comment:
Sorry, I know I should look this up, but shouldn't test env vars like these be defined in `phunit.xml.dist`? If not, no worries.

Also, it might be good to make these different values, and comprehensible, so people don't think they're meant to be identical or useful. e.g. `eyThisIsATestAccessToken`.

Reply:
Do they not work when added to `phpunit.xml.dist`?